cluster-autoscaler/estimator/binpacking_limit.go

```diff
@@ -0,0 +1,5 @@
+package estimator
+
+type BinpackingLimit interface {
+	GetLimit() int
```

Comment:
`GetLimit` is a bit ambiguous, I think `GetNodeLimit` would better show the intent.

Reply:
Makes sense - I'm renaming it